src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -977,15 +977,14 @@ OverlayImpl::relay (protocol::TMProposeSet& m,
 {
     if (m.has_hops() && m.hops() >= maxTTL)
         return;
-    std::set<Peer::id_t> skip;
-    if (! app_.getHashRouter().prepRelay (
-            uid, skip))
+    auto const skip = app_.getHashRouter().shouldRelay(uid);
+    if (! skip)
```

Comment:
Nit: The code reads a little weird. It looks like "if I don't want to skip, then return (thereby skipping)". Maybe don't use auto so it's clear the variable is an optional collection and not a bool, or consider renaming the variable so it sounds more like a collection and less like a bool. Maybe `toSkip`?

I understand the code, and it's correct, but consider renaming the variable.